classes/Upgrader.php

```diff
@@ -101,14 +101,11 @@ public function downloadLast($dest, $filename = 'prestashop.zip')
 
 		$destPath = realpath($dest).DIRECTORY_SEPARATOR.$filename;
 
-		if ($zip = Tools14::file_get_contents($this->link, false, null, 1000))
-		{
-			if((bool)file_put_contents($destPath, $zip) === true)
-				return true;
-		}
-		else
-			return false;
+        Tools::copy($this->link, $destPath);
```

Comment:
We prefer to use legacy methods to avoid to re-test the all module as it should be reworked "soon"